### PR TITLE
[DD-725] Remove 2.0 Path from Sidenav

### PR DESCRIPTION
**Pull request by tshen815**
# Description
We need to clean up content to remove all references to `2.0`

# Considerations
This PR is only removing the `2.0` path from the sidenav 

# Before
Screen Shot 2022-07-11 at 3 55 57 PM

# After
Screen Shot 2022-07-11 at 3 55 36 PM

# Design
@aaylacci Please ensure that the coloration of the highlight in the side nav is correct. As seen in the after picture.

### CODE REVIEW

File: jekyll/_data/sidenav.yml [hunk 1]
```diff
@@ -7,307 +7,310 @@ en:
...
 
 - name: Execution environments
   icon: icons/sidebar/executors.svg
   children:
     - name: Overview
       children:
         - name: Introduction
-          link: 2.0/executor-intro/
-        - name: Migrating Docker to Machine
-          link: 2.0/docker-to-machine/
```
Comment (rsylvian) [inline jekyll/_data/sidenav.yml:132]: why is this removed?

**Review by rsylvian — CHANGES_REQUESTED** [review]
It seems like some links disappeared and some sections got renamed? Any reasons for that? Should we we keep this PR strictly about removing `2.0`?

File: jekyll/_data/sidenav.yml [hunk 2]
```diff
@@ -316,334 +319,339 @@ en:
...
-          link: 2.0/databases/
+          link: /databases
         - name: Database Examples
-          link: 2.0/postgres-config/
+          link: /postgres-config
 
     - name: Testing
       children:
-        - name: Overview
-          link: 2.0/test/
```
[the remark this answers is not in the document]
Comment (chantelle11) [reply jekyll/_data/sidenav.yml:377]: Fixed

**Review by rsylvian — APPROVED** [review]
LGTM ✅

**Review by devinmarieb — APPROVED** [review]

File: jekyll/_config.yml [hunk 3]
```diff
@@ -98,8 +98,8 @@ collections:
     name: "2.0 Docs"
     site_section: true
     output: true
-    permalink: /2.0/:path/
```
Comment (rsylvian) [inline jekyll/_config.yml:101]: are we sure we don't need the slash in front?

Comment (chantelle11) [reply jekyll/_config.yml:101]: Everything seems to be working on my end

Comment (chantelle11) [reply jekyll/_config.yml:101]: I can add it back no worries and test

**Review by devinmarieb — APPROVED** [review]

**Review by Mayuko-Sa — APPROVED** [review]

File: jekyll/_cci2/persist-data.md [hunk 4]
```diff
@@ -57,8 +57,8 @@ If you notice your workspace usage is high and would like to reduce it, try sear
 
 You also might find that you are only using workspaces to be able to re-run builds from fail. Once the failing build passes, the workspace might not be needed. Setting a low storage period of, for example, one day, might be suitable for your projects. A low storage retention period for workspaces will save costs by not keeping unnecessary data in storage.
 
-* For more information on workspace optimization, configuration, and expiration, see the [Using Workspaces]({{site.baseurl}}/2.0/workspaces/) page.
-* For more information on workflows, see the [Workflows]({{site.baseurl}}/2.0/workflows/) page. 
+* For more information on workspace optimization, configuration, and expiration, see the [Using Workspaces]({{site.baseurl}}/workspaces/) page.
+* For more information on workflows, see the [Workflows]({{site.baseurl}}/workflows/) page.
```
[the remark this answers is not in the document]
Comment (rsylvian) [reply jekyll/_cci2/persist-data.md:61]: @chantelle11 Are you sure there is no fix to be done? Can we double-check with the Docs team, please?

[the remark this answers is not in the document]
Comment (chantelle11) [reply jekyll/_cci2/persist-data.md:61]: Verified with Devin its all good.
